index.js

```diff
@@ -37,5 +39,4 @@ app.put('/image', (req, res) => { image.imageHandler(req, res, db) });
 
 app.listen(process.env.PORT || 3005, () => {
     console.log(`listening on port${process.env.PORT}`)
-    console.log(process.env)
```

Comment:
This log was removed because I think it was useless and made the console very clouded! 🤔 what do you think about it?